data/campaigns/Eastern_Invasion/scenarios/14_The_Drowned_Plains.cfg

```diff
@@ -37,6 +37,7 @@
 
     [story]
         [part]
+            #po: In this scenario, there's circa 18 ghosts, each with a single line of speech, you'll recognise them because most start and end with ellipsis. The WML would need to be restructured to put po hints on those lines themselves.
```

Comment:
What? Why? Each one is on its own line, so why can't they have a po hint? Or in the worst case, you could add a line break just before the string and put the hint there?